offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -358,14 +358,14 @@ class QuotationDetails {
                     """
         }
 
-        static String discountItemInHTML(int offerPosition, String description, double discountAmount) {
+        static String discountItemInHTML(int offerPosition, String description, OfferItem offerItem) {
```

Comment:
There are lots of public methods here that do not have JavaDoc. Please add it for the methods you modified

Reply:
or can this be private?

Reply:
I think it can be private, I will have a look

Reply:
This is part of the private class ItemPrintout, so it can only be accessed from within the QuotationDetails class. Its not exposed to the outside.